react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+
+const topics = [
+  {
+    id: 1,
+    topic: 'Tastea',
+    heading: 'Tastea',
+    details: 'They have solid fruit teas and their boba is green. Their milk teas are not very tasty.Popcorn chicken and fries are great, but the portion size got smaller.'
+  },
+  {
+    id: 2,
+    topic: 'HappyLemon',
+    heading: 'Happy Lemon',
+    details: 'Don\'t get their boba. They specialize in cheese foam drinks. They also have bubble waffles which are pretty good.Sometimes they have crepecakes too.'
+  },
+  {
+    id: 3,
+    topic: 'SevenLeaves',
+    heading: '7 Leaves',
+    details: 'Boba is a hit or a miss.Sometimes it\'s too squishy and overcooked. Their taro milk tea is the best, but it could do with less sugar. Their menu has a good variety of teas.'
+  }
+];
+
+class Accordion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      topic: '',
+      display: false
+    };
+    this.handleClick = this.handleClick.bind(this);
+    this.hideDetails = this.hideDetails.bind(this);
+    this.showDetails = this.showDetails.bind(this);
+  }
+
+  hideDetails() {
+    const details = document.querySelector('#' + this.state.topic);
+    details.classList.add('display-none');
```

Comment:
We don't want to do anything that modifies the DOM directly. Instead, you can store _which_ topic to open. If it was already open, you can open "none".